manifests/f/FastCopy/IPMsg/5.0.3/FastCopy.IPMsg.yaml

```
@@ -0,0 +1,19 @@
+PackageIdentifier: FastCopy.IPMsg
+PackageVersion: 5.0.3
+PackageName: IP Messenger
+Publisher: FastCopy Lab, LLC.
+License: BSDL
+Moniker: ipmsg
+ShortDescription: Serverless lightweight Messenger for LAN.
+PackageUrl: https://ipmsg.org/
+InstallerType: exe
+Installers:
+- Architecture: x86
+  InstallerUrl: https://ipmsg.org/archive/ipmsg5.0.3_installer.exe
+  InstallerSha256: ca1ab217ff56d0bfa04bbcf2bfea978ffe64cfb35f40a5b4dc8c7b13bf927fab
+  InstallerSwitches:
+    Silent: "/SILENT"
+    SilentWithProgress: "/SILENT"
+PackageLocale: ja-JP
```

```suggestion
PackageLocale: en-US
```
As far as i can see its installed in English